sources/_base_sources/degen_base_sources.yml

```diff
@@ -0,0 +1,403 @@
+        columns:
+          - *block_date
+          - *block_time
+          - *block_number
+          - name: index
+            description: "Log index"
+          - name: contract_address
+            description: "Address of the degen smart contract generating the log"
+          - name: contract_name
+            description: "Name of the degen smart contract generating the log"
+          - name: event_name
+            description: "Name of the degen smart contract event generating the log"
+          - name: namespace
+            description: "Namespace of the degen smart contract generating the log"
+          - name: signature
+            description: "Signature of the event"
+          - *tx_hash
+
+      - name: blocks
+        freshness:
+          warn_after: { count: 1, period: day }
+        loaded_at_field: time
+        description: "Blocks are batches of transactions with a hash of the previous block in the chain. This links blocks together (in a chain) because hashes are cryptographically derived from the block data."
+        columns:
+          - name: base_fee_per_gas
+          - name: difficulty
+          - name: gas_limit
+          - name: gas_used
+          - name: hash
+          - name: miner
+          - name: nonce
+          - name: number
+          - name: parent_hash
+          - name: size
+          - name: time
+          - name: total_difficulty
+
+      - name: contracts
+        freshness:
+          warn_after: { count: 1, period: day }
+          error_after: { count: 3, period: day }
+        loaded_at_field: created_at
+        description: "A view keeping track of what contracts are decoded on Dune on Degen; contains information associated with the decoded contract such as namespace, name, address, ABI."
+        columns:
+          - name: abi
+            description: "ABI of the decoded contract"
+          - name: address
+            description: "Address of the decoded contract"
+          - name: from
+            description: "Address that created/deployed this contract"
+          - name: code
+            description: "Code for contract creation"
+          - name: name
+            description: "Name of the decoded contract"
+          - name: namespace
+            description: "Namesapce of the decoded contract"
+          - name: dynamic
+            description: "Boolean indicating whether the contract is dynamic or not"
+          - name: base
+            description: "Boolean indicating whether the contract is the base contract or not"
+          - name: factory
+            description: "Boolean indicating whether the contract is a factory contract or not"
+          - name: detection_source
+            description: "Detection source: 'factory', 'base', or 'dynamic'"
+            tests:
```

Comment:
these tests are failing because these sources don't exist for mode/degen, since there is no decoding on top of raw. can you go through these sources and remove the ones that don't exist?